### PR TITLE
v2 copy endpoints

### PR DESCRIPTION
**Pull request by baude**
add copy endpoint inputs and outputs.  these endpoints are not implemented yet, nor are any bindings.  this allows us to update this later without having to change our api version.

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by jwhonce** [comment]
LGTM Swagger verify tool expects one verb per endpoint.  It either needs a skip or update.

File: pkg/api/server/register_archive.go [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+	//      schema:
+	//       type: string
+	//       format: binary
+	//    400:
+	//      $ref: "#/responses/BadParamError"
+	//    404:
+	//      $ref: "#/responses/NoSuchContainer"
+	//    500:
+	//      $ref: "#/responses/InternalError"
+	r.HandleFunc(VersionedPath("/containers/{name}/archive"), s.APIHandler(compat.Archive)).Methods(http.MethodPost, http.MethodGet)
```
Comment (edsantiago) [inline pkg/api/server/register_archive.go:87]: The way I wrote my swagger-check was pretty sloppy: it only checks the first `http.MethodXxx`. If you reverse these two, so `MethodGet` is first, the test will pass.

File: pkg/api/server/register_archive.go [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+	//      schema:
+	//       type: string
+	//       format: binary
+	//    400:
+	//      $ref: "#/responses/BadParamError"
+	//    404:
+	//      $ref: "#/responses/NoSuchContainer"
+	//    500:
+	//      $ref: "#/responses/InternalError"
+	r.HandleFunc(VersionedPath("/libpod/containers/{name}/copy"), s.APIHandler(libpod.Archive)).Methods(http.MethodPost, http.MethodGet)
```
Comment (edsantiago) [inline pkg/api/server/register_archive.go:167]: Same thing (put `MethodGet` first)

**Comment by edsantiago** [comment]
Via `cirrus-flake-summarize`. That json flake is really hurting us.

#### test fedora-31 fedora-31

  2020-05-26T11:54:18  [integration_test](https://api.cirrus-ci.com/v1/task/5811306894721024/logs/integration_test.log)
*   [Podman build [It] podman build basic alpine and print id to external file ](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/5811306894721024/html/integration_test.log.html#t--podman-build-basic-alpine-and-print-id-to-external-file--1)
*   [Podman rmi [It] podman rmi image that is a parent of another image ](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/5811306894721024/html/integration_test.log.html#t--podman-rmi-image-that-is-a-parent-of-another-image--1)
*   [Podman stats [It] podman stats with json output ](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/5811306894721024/html/integration_test.log.html#t--podman-stats-with-json-output--1)
  *   [[same]](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/5811306894721024/html/integration_test.log.html#t--podman-stats-with-json-output--2)
  *   [[same]](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/5811306894721024/html/integration_test.log.html#t--podman-stats-with-json-output--3)

  2020-05-26T12:37:25  [integration_test](https://api.cirrus-ci.com/v1/task/4879691041472512/logs/integration_test.log)
*   [Podman stats [It] podman stats with json output ](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/4879691041472512/html/integration_test.log.html#t--podman-stats-with-json-output--1)
  *   [[same]](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/4879691041472512/html/integration_test.log.html#t--podman-stats-with-json-output--2)
  *   [[same]](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/4879691041472512/html/integration_test.log.html#t--podman-stats-with-json-output--3)

  2020-05-26T13:20:20  [integration_test](https://api.cirrus-ci.com/v1/task/6086469980782592/logs/integration_test.log)
*   [Podman login and logout [It] podman login and logout with flag --authfile ](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/6086469980782592/html/integration_test.log.html#t--podman-login-and-logout-with-flag---authfile--1)
  *   [[same]](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/6086469980782592/html/integration_test.log.html#t--podman-login-and-logout-with-flag---authfile--2)
*   [Podman login and logout [It] podman login and logout with --tls-verify ](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/6086469980782592/html/integration_test.log.html#t--podman-login-and-logout-with---tls-verify--1)
*   [Podman login and logout [It] podman login and logout with flag --authfile ](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/6086469980782592/html/integration_test.log.html#t--podman-login-and-logout-with-flag---authfile--3)
*   [Podman login and logout [It] podman login and logout with --tls-verify ](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/6086469980782592/html/integration_test.log.html#t--podman-login-and-logout-with---tls-verify--2)
  *   [[same]](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/6086469980782592/html/integration_test.log.html#t--podman-login-and-logout-with---tls-verify--3)
*   [Podman stats [It] podman stats with json output ](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/6086469980782592/html/integration_test.log.html#t--podman-stats-with-json-output--1)
  *   [[same]](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/6086469980782592/html/integration_test.log.html#t--podman-stats-with-json-output--2)
  *   [[same]](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/6086469980782592/html/integration_test.log.html#t--podman-stats-with-json-output--3)
*   [Podman network [It] podman network rm ](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/6086469980782592/html/integration_test.log.html#t--podman-network-rm--1)

**Comment by vrothberg** [comment]
I opened https://github.com/containers/libpod/pull/6406 for the `podman stats` flake.

**Pull request merged**